### ISSUE TITLE
Custom Licence name returns null for requests made in a locale different from the original set-up  

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-server#9963]
Similar to https://github.com/mozilla/addons-server/issues/7535

STR:
1. Submit a new add-on while your locale is set to `en-GB`
2. In the submission details flow, add a Custom Licence with a name of your choice
3. After the add-on is auto-approved, get the api details of your add-on in `en-US`, i.e https://addons-dev.allizom.org/api/v4/addons/addon/{your-addon}/?lang=en-US
4. Check the  `"license": {..."name"` value

Actual result:
The license name is `null`

Expected result:
The license `name` should contain the value given by you during submission, regardless of the locale in which the request was made

Notes:
- example https://addons-dev.allizom.org/api/v4/addons/addon/greatly-for-firefox/?lang=en-US

![image](https://user-images.githubusercontent.com/31961530/48345013-73a18380-e67f-11e8-82e6-856685e26232.png)

As seen on the frontend:

![image](https://user-images.githubusercontent.com/31961530/48345123-caa75880-e67f-11e8-95a6-5c5410624dd2.png)

**Comment by diox** [comment]
Yes, this is not a new issue. The root cause is similar to https://github.com/mozilla/addons-server/issues/4599 - the model doesn't have a `default_locale`, and if it did it wouldn't necessarily follow the add-on's `default_locale` either, so that messes up cases like these.

**Comment by AlexandraMoga** [comment]
This is still a valid issue

**Comment by AlexandraMoga** [comment]
Still an issue

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-172

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-172
